src/asInput/index.jsx

```diff
@@ -149,21 +163,28 @@ const asInput = (WrappedComponent, labelFirst = true) => {
     render() {
       const { description, error, describedBy } = this.getDescriptions();
       return (
-        <div className={styles['form-group']}>
-          {labelFirst && <label id={`label-${this.state.id}`} htmlFor={this.state.id}>{this.props.label}</label>}
+        <div className={[classNames({
+            [styles['form-group']]: inputType !== 'checkbox',
+            [styles['form-check']]: inputType === 'checkbox',
+          })]}
```

Comment:
Bootstrap documentation shows that checkboxes should be wrapped with the form-check style, the CheckBoxGroup will wrap multiple checkboxes within the same group (instead of each in their own group) as expected. https://getbootstrap.com/docs/4.0/components/forms/#checkboxes-and-radios